### ISSUE TITLE
Update Compact Solars textures to match Unity's IC2 textures

### ISSUE THREAD

**Issue by LuigiHutch** [CyanideX/Unity#75]
As it seems that Cyanide is on an IC2 texturing theme (see recent IC2 + IC2 classic retexture), do you have any pans to update the Compact Solars retexure to closely match the IC2 retexture style.

Currently the ones in the repo match the current (vanilla) compact solars texture and do not match the style of the IC2 retexture. Also as an added feature request, could retexture the solar hats as well while your at it ;)

Thanks
 - LuigiHutch

**Comment by CyanideX** [comment]
Yes, of course! I'm just running low on free time at the moment.

**Comment by AFR0N1NJAZ** [comment]
I've been working on the Compact Solars textures, I'm almost done with them. If CyanideX could approve them they would be good to go.

**Comment by CyanideX** [comment]
Post a screenshot here @AFR0N1NJAZ. 😄 I'll let you know what I think.

**Comment by AFR0N1NJAZ** [comment]
![2016-10-07_16 15 01](https://cloud.githubusercontent.com/assets/11052983/19204426/c9b2d8f4-8caa-11e6-82cf-3fe0fac4a53e.png)

**Comment by AFR0N1NJAZ** [comment]
Thinking about changing the LV Panel.

**Comment by CyanideX** [comment]
Those look very similar to the ones I made for IC2 Classic; however, following the design style from the basic solar panels, you should only have the "ports" on the bottom face. The sides should all be the same texture. I'm not too keen on the top having alternate textures. Altogether I think you've done a good job at pulling in the the elements from other textures. :)

**Comment by AFR0N1NJAZ** [comment]
Thank you. I will take your advice.

**Comment by CyanideX** [comment]
This is an example of the ones I did for IC2 Classic:

![image](http://i.imgur.com/xF0Y5qi.png)

**Comment by AFR0N1NJAZ** [comment]
That's funny I didn't even know.
